content/docs/get-started/index.md

```diff
@@ -21,184 +23,142 @@ $ mkdir mlem-get-started
 ```
 
-We'll use docker later on to package and serve a model locally. To install
-docker, please refer to the
-[official installation guide](https://docs.docker.com/get-docker/).
+- We'll use docker later on to package and serve a model locally. To install
+  docker, please refer to the
+  [official installation guide](https://docs.docker.com/get-docker/).
+- In the [last section](#deploying-models-in-production) of the guide, we will
+  mention additional pip packages which will be needed per our deployment type
+  (e.g. `mlem[kubernetes]` to deploy on a Kubernetes cluster).
 
 That's it, it's that simple! You're ready to MLEM.
 
 </details>
 
-## Saving your model
+## Saving models
+
+To unlock MLEM's power, we need to capture model metadata that is not usually
+captured when models are saved.
+
+Let's say we want to save a model to the path `models/rf`. This is usually done
+with a framework helpers like `torch.save()`, or better yet, libraries like
+`pickle` or `joblib`.
+
+<details>
 
-Before we see how many things MLEM can do for us, we first need to save an ML
-model to a file with MLEM.
+### Here's a typical example of training code
 
-As a basic example, create and execute the following `train.py` Python script:
+Consider the following `train.py`:
 
 ```py
 from sklearn.datasets import load_iris
 from sklearn.ensemble import RandomForestClassifier
+import joblib
 
-from mlem.api import save
-
-
+# load data
 data, y = load_iris(return_X_y=True, as_frame=True)
+
+# instantiate and train model
 rf = RandomForestClassifier(
     n_jobs=2,
     random_state=42,
 )
 rf.fit(data, y)
-save(
-    rf,
-    "models/rf",
-    sample_data=data,
-)
+
+# save to file
+joblib.dump(rf, "models/rf")
 ```
 
-Here, we loaded a well-known
+Here We loaded a well-known
 [Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-`scikit-learn` and trained a simple classifier. Instead of pickling the model,
-we persisted it using MLEM's `save` API.
+`scikit-learn` and trained a simple classifier. Note that `scikit-learn` is just
+an example of many MLEM supported ML frameworks (
+[full list here](/doc/object-reference/model)). Finally, we used `joblib` to
+serialize it to file, so executing the script will dump the model to
+`./models/rf`.
 
-`scikit-learn` is just an example of many supported ML frameworks. Check out the
-[full list here](/doc/object-reference/model).
+</details>
 
-Now, let's see what MLEM saved by using the `tree` terminal command:
+MLEM can analyze the code (or model files), and extract extra metadata which
+will become very useful when using the model.
 
-```cli
-$ tree models/
-models
-├── rf
-└── rf.mlem
+We can either save the model with MLEM to begin with, or use MLEM to import
+existing model file and save the metadata as a side effect.
+
+<toggle>
+<tab title="Save new models">
+
+Just replace the line saving the model with a call to `mlem.api.save`:
+
+```py
+from mlem.api import save
+...
```

Comment:
```suggestion
```
I'd remove - doesn't change the meaning.

Reply:
It looks awkward to me without it honestly, it signifies we know we refer to a specific line after (and probably before) a lot of the user's code